Mathlib/Logic/Relator.lean

```diff
@@ -32,7 +32,7 @@ def LiftFun (f : α → γ) (g : β → δ) : Prop :=
   ∀⦃a b⦄, R a b → S (f a) (g b)
 
 /-- `(R ⇒ S) f g` means `LiftFun R S f g`. -/
```

Comment:
Off-topic, but this is a very unhelpful docstring. It'd probably be better to just replace it by `@[inherit_doc]` and mention the notation in `LiftFun`'s docstring.

Reply:
I'd be happy to merge a PR with a docstring improvement!